src/test/scala/sbtorgpolicies/arbitraries/ValidationArbitraries.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sbtorgpolicies.arbitraries
+
+import cats.data.ValidatedNel
+import cats.syntax.validated._
+import org.scalacheck.{Arbitrary, Gen}
+
+trait ValidationArbitraries {
+
+  implicit def invalidNelGen[E, A](implicit E: Arbitrary[E]): Gen[ValidatedNel[E, A]] = {
```

Comment:
Nothing wrong with these but `cats` should already include arbitraries for all its datatypes including `Validated[NonEmptyList[E], A]` https://github.com/typelevel/cats/blob/5c00fdfdb361afb9660e79519b70804c0a0f074e/laws/src/main/scala/cats/laws/discipline/Arbitrary.scala#L65